docs/native/src/testing/testcases.rst

```diff
@@ -1,3 +1,140 @@
+..  _Manual Validation:
+Manual Validation
+======================
+Your WOPI integration can be tested following :ref:`validator app testing <Validator app>`.
+However, the validator app will not be able to test the complete end to end experience. You must run the following tests manually to verify the integration works correctly. 
 
-|stub-icon| Partner test cases
-==============================
+Please download the :download:`Testing <ManualTestPass.docx>` document and fill it in as you perform your testing. After both the validator and manual tests are both 100% passing, please provide Microsoft with a link to your validator app, your test account information and manual test results document. Microsoft will complete a final test pass before launch. 
+
+Please contact Microsoft for directions to set the Office for iOS apps into a test mode that can be used for testing the below scenarios. 
+
+
+ ..  note::
+	 * "Company app/application" refers to your app. 
+	 * "Company service" refers to your service. 
+	 * Where it refers to "Office", please substitute Word, Excel and PowerPoint app. The tests should be re-run against each Office application. 
+	 * Please note the version of the Office app you tested against. This information can be found under Settings -> [App] -> Version
+
+.. |Duplicate| image:: ../images/PartnerTestCases_Duplicate.png  
+    :alt: A screenshot that shows the document actions in Office for iOS. 
+.. |ImageProps1| image:: ../images/PartnerTestCases_Properties1.png 
+    :alt: A screenshot that shows document properties in the Office outspace. 
+.. |ImageProps2| image:: ../images/PartnerTestCases_Properties2.png
+    :alt: A screenshot that shows the document properties within the opened file. 
+
+-------------------------------------
```

Comment:
Don't use overlines. See here for the proper heading style hierarchy:

https://wopi.readthedocs.io/en/latest/contributing/style_guide.html#heading-styles